packages/edit-post/src/components/sidebar/sidebar-header/style.scss

```diff
@@ -33,3 +33,11 @@
 		}
 	}
 }
+
+.edit-post-sidebar-header__container {
+	display: flex;
+	flex-direction: column;
+	position: sticky;
+	top: 0;
+	z-index: z-index(".components-panel__header");
```

Comment:
I think the convention is that the z-index name should follow the selector name (even if we have to duplicate z-index value for multiple selectors.

Reply:
Updated